Content.Server/Resist/CanEscapeInventoryComponent.cs

```diff
@@ -8,4 +10,9 @@ public sealed class CanEscapeInventoryComponent : Component
     /// </summary>
     [DataField("baseResistTime")]
     public float BaseResistTime = 5f;
+
+    [DataField("isEscaping")]
+    public bool IsEscaping;
```

Comment:
unless do-afters support saving & loading from maps, this shouldn't be a data field